src/infrastructure/figma/figma-client/figma-client.ts

```diff
@@ -192,6 +194,28 @@ export class FigmaClient {
 
 		return response.data;
 	};
+
+	/**
+	 * Creates a new Figma webhook using the POST webhook endpoint
+	 *
+	 * @see https://www.figma.com/developers/api#webhooks-v2-post-endpoint
+	 */
+	createWebhook = async (
+		request: CreateWebhookRequest,
+		accessToken: string,
+	): Promise<CreateWebhookResponse> => {
+		const response = await axios.post<CreateWebhookResponse>(
+			`${getConfig().figma.apiBaseUrl}/v2/webhooks`,
```

Comment:
**nitpick:** While testing other webhook-related changes, I noticed that we need the `webhooks:write` scope to call this API.

We need to add this scope to [config.figma.scope](https://github.com/atlassian-labs/figma-for-jira/blob/2b04d0e6dd4d2d6c08790f663a91b2def7555e6a/src/config/config.ts#L48).

Reply:
Great catch, I totally missed this. I'll make sure to e2e test this before it gets merged.